### DISCUSSION TITLE
Scope of OpenRadioss license

### DISCUSSION THREAD

**Discussion by JJJFEM** [OpenRadioss/OpenRadioss#818] [Q&A] [answered]
Good night,

I want to know the scope of the licese, in terms to know if is possible to develop professional consultance (commercial use) and research with OpenRadioss software (of course, the idea is to use open pre and post-processors also). Thank you very much for your attention: regards

**Comment by MarianBulla** [comment, marked as answer]
Good morning,
Yes, from the license point of view [AGPL-3.0 license](https://github.com/OpenRadioss/OpenRadioss/blob/main/LICENSE.md) it is possible, as long as you do not modify the code. If you do modifications, you have to release them, too. Then everything is fine.

**Reply by JJJFEM** [reply]
¡Thank you!
